examples/chat/Cargo.toml

```diff
@@ -7,7 +7,3 @@ license = "GPL-3.0"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive"] }
-
-[[bin]]
-name = "encoder"
-path = "src/encode.rs"
```

Comment:
Should this file also be removed? If I don't miss anything it is not used anywhere.

Sorry for the late comment )